Assets/Scripts/Models/Character.cs

```diff
@@ -874,8 +876,60 @@ public IEnumerable<ContextMenuAction> GetContextMenuActions(ContextMenu contextM
         yield return new ContextMenuAction
         {
             Text = "Poke "+GetName(),
-            RequiereCharacterSelected = false,
+            RequireCharacterSelected = false,
             Action = (cm, c) => Debug.ULogChannel("Character", GetDescription())
         };
     }
+
+    public void PrioritizeJob(Job j)
+    {
+        AbandonJob(false);
+        World.current.jobQueue.Remove(j);
+        j.IsBeingWorked = true;
+        //check if the character is carrying any materials and if they could be used for the new job,
+        //if the character is carrying materials but is not used in the new job, then drop them
+        //on the current tile for now.
+        if (inventory != null && !j.inventoryRequirements.ContainsKey(inventory.objectType))
+        {
+            World.current.inventoryManager.PlaceInventory(CurrTile, inventory);
+            DumpExcessInventory();
+        }
+
+        myJob = j;
+
+        // Get our destination from the job
+        DestTile = myJob.tile;
+
+        // If the dest tile does not have neighbours that are walkable it's very likable that they can't be walked to
+        if (DestTile.GetNeighbours().Any((tile) => { return tile.MovementCost > 0; }) == false)
```

Comment:
I gues this could be writen as:
`if (DestTile.GetNeighbours().Any(tile => tile.MovementCost > 0) == false)`
or
`if (!DestTile.GetNeighbours().Any(tile => tile.MovementCost > 0))`